CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+
+## Coding Style
+
+The code in this package is also automatically formatted by `black` for consistency.
+
+## The Development Life Cycle
+
+<!--
+### Fork and clone this repository
+
+1. See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
+1. Then, [clone the repository](https://help.github.com/articles/cloning-a-repository/) to your local machine so you can begin making changes.
+1. Add this repository as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) on your local git repository so that you are able to fetch the latest commits.
+1. On your local machine make sure you have the latest version of the `develop` branch:
+
+    ```shell
+    git checkout develop
+    git pull upstream develop
+    ``` -->
+
+### Install development dependencies
+
+Please follow the [README.md](README.md) to install the package for development purposes.
+
+### Developing at Sage Bio
+
+The agora-data-tools project follows the standard [trunk based development](https://trunkbaseddevelopment.com/) development strategy.
+> To ensure the most fluid development, do not push to your `dev`
+
+1. Please ask for write permissions to contribute directly to this repository.
+1. Make sure you are always creating feature branches from the `dev` branch.
+
+    ```shell
+    git checkout dev
+    git pull
+    ```
+
+1. Create a feature branch from the `dev` branch. Use the Id of the JIRA issue that you are addressing and name the branch after the issue with some more detail like `{user}/{JIRA}-123/add-some-new-feature`.
+
+    ```shell
+    git checkout dev
+    git checkout -b tyu/JIRA-123/some-new-feature
+    ```
+
+1. At this point, you have only created the branch locally, you need to push this to your fork on GitHub.
+
+    ```shell
+    git push --set-upstream origin tyu/JIRA-123/some-new-feature
+    ```
+
+    You should now be able to see the branch on GitHub. Make commits as you deem necessary. It helps to provide useful commit messages - a commit message saying 'Update' is a lot less helpful than saying 'Remove X parameter because it was unused'.
+
+    ```shell
+    git commit changed_file.txt -m "Remove X parameter because it was unused"
+    git push
+    ```
+
+1. (Make sure you have follow instructions in "Install development dependencies") Once you have made your additions or changes, make sure you write tests and run the test suite.  More information on testing below.
```

Comment:
Typo: follow**ed** instructions.
Can we link to the relevant page/section that corresponds to "Install development dependencies"? Is this in the Readme.md file? Also it doesn't make sense why this comment is down here right before the section on testing, and not up at the top with the Readme info.